AppSource/CUDALaplacePropagation.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <cstdio>
+#include <cstdint>
+#include <vector>
+
+#include "cuda_runtime.h"
+#include "device_launch_parameters.h"
+
+class CUDALaplacePropagation
```

Comment:
Replace with namespace. Same syntax, but does not indicate storing any state. 
Than you can use 

`using namespace CUDAHelpers::CUDALaplacePropagation;` in *.cu files